package.json

```diff
@@ -2,60 +2,67 @@
   "name": "loader339",
   "version": "1.0.0",
   "description": "ES Module Loader Prototype — APM",
+  "keywords": [],
+  "homepage": "https://github.com/DerekNonGeneric/loader339#readme",
+  "bugs": {
+    "url": "https://github.com/DerekNonGeneric/loader339/issues"
+  },
+  "license": "0BSD",
```

Comment:
```suggestion
  "license": "MIT",
```

<br>

Just as a heads up, [Socket](https://socket.dev) most strongly weight the [Blue Oak License Council](https://blueoakcouncil.org)'s opinion of licenses since they seem most in line with the concerns of package users. They categorize the MIT license as &ldquo;Silver&rdquo; while [BSD-2-Clause Plus Patent License](https://spdx.org/licenses/BSD-2-Clause-Patent.html) (`BSD-2-Clause-Patent`) is their only &ldquo;Gold&rdquo; one. I am a bit torn, but if pretty much everyone is using MIT and they are requiring (sometimes via automated tooling) that their deps also be MIT as well, I am not too sure we have many options here. 🤔

Refs: https://blueoakcouncil.org/list#gold